package.json

```diff
@@ -12,7 +12,8 @@
     "clean": "./node_modules/.bin/lerna run clean --stream",
     "build": "./node_modules/.bin/lerna run build --stream",
     "serve": "./node_modules/.bin/http-server -a 127.0.0.1 -o /packages/modelviewer.dev -c-1",
-    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream"
+    "update:package-lock": "rm -f ./package-lock.json && npm i && ./node_modules/.bin/lerna run update:package-lock --stream",
+    "upgrade": "rm -rf node_modules package-lock.json && npm install && lerna exec 'rm -rf node_modules package-lock.json' && lerna bootstrap && npm run clean"
```

Comment:
I am still getting `npm WARN old lockfile The package-lock.json file was created with an old version of npm` when I run update:package-lock from your branch. Do you get that if you pull it to your other machine?

Reply:
Sounds like you're not seeing that if you run upgrade?

Reply:
Indeed, let's simplify so we only include the working script, under the old name.

Reply:
Just sent an updated version